data_pipeline/utils/data_store/bq_data_service.py

```diff
@@ -466,7 +466,7 @@ def load_given_json_list_data_from_tempdir_to_bq(
     project_name: str,
     dataset_name: str,
     table_name: str,
-    json_list: str
+    json_list: list
```

Comment:
actually it should accept an iterable

Reply:
In practice this will usually be `Iterable[dict]`. But in theory it could have other types. So maybe `Iterable[Any]` would be more accurate.

Reply:
OK!